Source/WebCore/bindings/js/WorkerModuleScriptLoader.cpp

```diff
@@ -89,7 +89,7 @@ void WorkerModuleScriptLoader::load(ScriptExecutionContext& context, URL&& sourc
     bool cspCheckFailed = false;
     ContentSecurityPolicyEnforcement contentSecurityPolicyEnforcement = ContentSecurityPolicyEnforcement::DoNotEnforce;
     if (!context.shouldBypassMainWorldContentSecurityPolicy()) {
-        auto* contentSecurityPolicy = context.contentSecurityPolicy();
+        CheckedPtr contentSecurityPolicy = context.contentSecurityPolicy();
```

Comment:
Should `ScriptExecutionContext` have a `checkedContentSecurityPolicy()` accessor?

Reply:
It does have a `checkedContentSecurityPolicy()` but I still prefer my way.